pkg/algorithm/topology_aware_scheduler.go

```diff
@@ -204,7 +205,15 @@ func (cv clusterView) Len() int {
 }
 
 func (cv clusterView) Less(i int, j int) bool {
-	if cv[i].usedGpuNumSamePriority > cv[j].usedGpuNumSamePriority {
+	if !cv[j].healthy {
+		return true
+	} else if !cv[i].healthy {
+		return false
+	} else if !cv[j].suggested {
```

Comment:
No need !, such as :

 if cv[i].healthy {
  return true
 } else if cv[j].healthy {
  return false
 }


And comment the sorting rule above the func.